mne/viz/utils.py

```diff
@@ -1297,3 +1297,80 @@ def _compute_scalings(scalings, inst):
         scale_factor = np.max(np.abs(scale_factor))
         scalings[key] = scale_factor
     return scalings
+
+
+class DraggableColorbar(object):
+    """Class for enabling interactive colorbar.
+    See http://www.ster.kuleuven.be/~pieterd/python/html/plotting/interactive_colorbar.html
```

Comment:
It's under Creative Commons Attribution 3.0 License. See http://www.ster.kuleuven.be/~pieterd/python/html/index.html